generators/app/templates/build.gradle

```diff
@@ -3,7 +3,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'org.xtext:xtext-gradle-plugin:2.0.2'
+        classpath 'org.xtext:xtext-gradle-plugin:2.0.8'
```

Comment:
2.0.8 still seems quite old. i guess also the gradle version
and rest of gradle file needs update, also jcenter should be removed

Reply:
Yes, I was saving that for a future PR since it's a pretty big jump (Gradle 5.1.1 to 7.4.2) and I haven't gotten everything to work yet. But since there seems to be low activity around this project I can take some more time and include the Gradle update in this PR as well.

Reply:
ok, i assume you can check what xtext new project wizard in eclipse creates as a base.

Reply:
Yes, looking into that now. Converting to draft in the meantime. Thank you!